### ISSUE TITLE
Proxmox Build

### ISSUE THREAD

**Issue by BabyDaddyGamer** [rapid7/metasploitable3#389]
I have successfully built both boxes using Win10 and Virtual Box.
I have successfully ported the Ubuntu box to Proxmox and it runs well.

My problem is that when I port the Windows box and boot it, it only boots to system recovery mode. It recognizes that the build is there due to having to log in with the vagrant credentials but none of the standard Windows Server fixes work, such as , sfc /scannow, chkdsk /r, and the automated startup recovery. The only thing that was found was that the boot wanted VBoxGuest.sys which it already had and when replacing it with the file from the original build it caused a BSOD.

Any suggestions?

**Comment by r-joyce** [comment]
When is it exactly BSODing? When I initially ported the VM into Proxmox and started it, it was BSOD at boot and would enter into windows recovery. I changed the HDD from SCSI to IDE and added a NIC as a VIRTIO type and now it seems to boot properly. How did you add the HDD to the VM?

The steps I took to get it into Proxmox were as followed:
1. Add the vagrant box with the VirtualBox provider
    `vagrant box add rapid7/metasploitable3-win2k8`
2. Copied over the `metasploitable3-win2k8-disk001.vmdk` and `box.ovf` from the hidden vagrant directory in the home folder over to Proxmox.
3. On Proxmox, I created the VM using:
   `qm importovf <vmid> box.ovf local-lvm`
4. Imported the HDD into the new VM:
   `qm importdisk <vmid> metasploitable3-win2k8-disk001.vmdk local-lvm -format qcow2`
5. In the Proxmox GUI, updated the hardware to actually use the HDD as an IDE drive, add the virtualized NIC (VirtIO), and under options I changed the OS Type to "Microsoft Windows 7/2008r2"

Hope that helps
